narps_open/utils/description/analysis_pipelines_full_descriptions.tsv

```diff
@@ -441,7 +441,7 @@ Custom-built MATLAB (2018b) scripts were used for extracting the white matter PC
+6) Spatial smoothing (5mm FWHM) and removal of first 10 volumes"	We used the tool BET in FSL for brain extraction. The fractional intensity threshold was set at 0.3 instead of the default value 0.5, so that it's less likely to remove valid brain voxels. 	We used the tool FAST in FSL for segmentation with the default parameters.  	We did not perform slice time correction.	We used the tool MCFLIRT in FSL for volume realignment. The reference scan was the middle scan. We used the default parameters of MCFLIRT.		We did not perform gradient distortion correction.	We used the tool FLIRT in FSL for function to structure coregistration. This is a linear transformation with 6 degrees of freedom. We used the default parameters of FLIRT.	We did not perform distortion correction.	We used FSL FLIRT for functional to structural registration (T1), followed by FSL FNIRT for structural to template registration (MNI space, 2mm) with 10 mm warp resolution and 12 degrees of freedom (normal search).	We did not perform intensity correction.	We used FSL for our analysis, thus each run was normalized such that the mean image had an intracerebral mode of 10,000.	We used the average time-course of the white matter as a counfound regressor, as well as a selection of PCA components extracted from the white matter compartments. Specifically, for each PCA component, a linear regressor analysis was performed with the PCA component timeseries as the output and 12 motion regressors (6 motion parameter timeseries extracted from volume realignment plus their 1st temporal derivatives) as explanatory variables. PCA components whose model yielded correlation coefficient above 65% were considered as nuisance regressors in the first-level analysis. Overall, this analysis returned between 5 and 15 nuisance regressors for each run. Typically, higher number of nuisance regressors was linked to higher mean framewise displacement.	We did not perform volume censoring.	We used FSL Feat to perform spatial smoothing, with a 5mm FWHM Gaussian kernel. The smoothing was performed in the native volume.	NA	"The number of points submitted to statistical modeling was 443 points, since we removed the first 10 volumes at the pre-processing stage to allow steady state magnetization.
```

Comment:
Why did this change ?

Reply:
see another comment for a complete list of changes to this file

Reply:
no longer pertinent -- file is now ignored